GravatarApp/AvatarPicker/AvatarUpload/ImagePickerSectionView.swift

```diff
@@ -0,0 +1,62 @@
+import SwiftUI
+
+struct ImagePickerSectionView: View {
+    let onImageSelected: (UIImage) -> Void
+
+    var body: some View {
+        VStack(alignment: .leading, spacing: 16) {
+            VStack(alignment: .leading, spacing: 0) {
+                Text("Get a new look")
+                    .font(.headline)
+                Text("It’s been 87 days since you updated your avatar.")
```

Comment:
We don't have localization yet but I think it would still be good to define these in a way that is easy for Glotpress to pick up in the furue, which is by declaring them as NSLocalizedString. WDYT?